src/handlebars/getinvolved.handlebars

```diff
@@ -21,25 +21,19 @@
   </div>
 
   <div class="margin-bottom">
-    <h2 class="bold">I'm an OpenJDK Developer</h2>
+    <h2 class="bold">I'm an Eclipse OpenJ9 and/or OpenJDK Developer</h2>
```

Comment:
As software developers, we should know that binary OR includes AND 😏

Reply:
Fixed :-D